ivre/utils.py

```diff
@@ -312,7 +312,7 @@ def str2regexp(string):
 
     """
     if string.startswith('/'):
-        string = string.split('/', 2)[1:]
+        string = string[1:].rsplit('/', 1)
```

Comment:
If we want to accept `/` in regexp, they should be escaped. We probably want to replace `\/` by `/`.